src/validate-parameters.coffee

```diff
@@ -3,9 +3,12 @@ validateParameters = (params) ->
   result = {warnings: [], errors: []}
 
   for paramName, param of params
-    if param.required is true and not param.example
-      text = "Required URI parameter '#{paramName}' has no example value."
-      result.errors.push(text)
+    if param.required is true and
+      (param.example is '' or param.example is undefined or param.example is null) and
+      (param.default is '' or param.default is undefined or param.default is null)
```

Comment:
why not just `if param.required and not param.example and not param.default`? are there any corner cases?

<img width="286" alt="screen shot 2016-07-22 at 09 00 20" src="https://cloud.githubusercontent.com/assets/283441/17049041/d6fc4164-4fea-11e6-881a-68a71967c931.png">